java/ql/lib/semmle/code/java/security/XxeQuery.qll

```diff
@@ -0,0 +1,55 @@
+/** Provides taint tracking configurations to be used in XXE queries. */
+
+import java
+import semmle.code.java.dataflow.FlowSources
+import semmle.code.java.dataflow.TaintTracking
+import semmle.code.java.dataflow.TaintTracking2
+import semmle.code.java.security.XmlParsers
+import DataFlow::PathGraph
```

Comment:
Let's not put top-level `query` predicates outside `.ql` files, I think.
```suggestion
```

Reply:
Yeah, copy paste error, thank you and Code Scanning for spotting it!